src/daemon.c

```diff
@@ -119,7 +119,7 @@ int main(int argc, char *argv[])
 	pid_t pid;
 	int rc = SUCCESS, i;
 	pthread_t mgmt_tid = INVALID_VALUE;
-	unsigned char path[NAMELEN] = {0};
+	unsigned char *path;
```

Comment:
Maybe:

```C
unsigned char *path = NULL;
```

Just to be sure...

Reply:
Ah, yes! I usually do that and forgot this time. Thank you! That is why we do these reviews. :-)